src/mono/msbuild/apple/build/AppleBuild.LocalBuild.props

```diff
@@ -22,9 +22,9 @@
   <Import Project="$(MSBuildThisFileDirectory)AppleBuild.props" />
 
   <PropertyGroup>
-    <_NetCoreAppCurrent>net8.0</_NetCoreAppCurrent>
+    <_NetCoreAppCurrent>net9.0</_NetCoreAppCurrent>
     <TargetFramework>$(_NetCoreAppCurrent)</TargetFramework>
-    <_NetCoreAppToolCurrent>$(_NetCoreAppCurrent)</_NetCoreAppToolCurrent>
+    <_NetCoreAppToolCurrent>net8.0</_NetCoreAppToolCurrent>
```

Comment:
Please add TODO comment here to indicate that this should be updated when a .NET 9 SDK is consumed that supports the net9.0 TFM.